docker-compose.yml

```
@@ -7,7 +7,16 @@ services:
       - 8080:8080
     env_file:
       - .env
+    environment:
+      - PORT=8080
+      - QUEUE_URL=amqp://rabbitmq
     volumes:
       - .:/app
+  rabbitmq:
+       image: rabbitmq:3.13-management
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 7

(indentation)